### PR TITLE
work on adaptivity (70%)

### PR DESCRIPTION
**Pull request by danil-react**
work on adaptivity

### CODE REVIEW

File: src/components/Card/styles.module.scss [hunk 1]
```diff
@@ -1,23 +1,155 @@
...
-  flex-direction: row;
   flex-wrap: wrap;
-  align-content: stretch;
-  align-items: stretch;
-  width: auto;
-  height: auto;
-
+  padding-left: 50px;
+  margin-right: 50px;
+  margin-left: 50px;
```
Comment (justiceteam-it) [inline src/components/Card/styles.module.scss:9]: margin: 0 50px;

File: src/components/Card/styles.module.scss [hunk 2]
```diff
@@ -1,23 +1,155 @@
...
+  padding-left: 50px;
+}
+
+@media screen and (max-width: 2560px) {
+  .card {
+    display: flex;
+    flex-wrap: wrap;
+    padding-left: 1px;
+    margin-right: 1px;
+    margin-left: 1px;
```
Comment (justiceteam-it) [inline src/components/Card/styles.module.scss:30]: margin:0 1px;

File: src/components/Card/styles.module.scss [hunk 3]
```diff
@@ -1,23 +1,155 @@
...
+}
+
+@media screen and (max-width: 1440px) {
+  .card {
+    display: flex;
+    flex-wrap: wrap;
+    padding-left: 50px;
+    margin-right: 50px;
+    margin-left: 50px;
+  }
```
Comment (justiceteam-it) [inline src/components/Card/styles.module.scss:50]: same

File: src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -1,29 +1,51 @@
...
-                        <Link to="/SignUp">SignUp</Link>
-                        <Link to="/SignIn">SignIn</Link>
-                        <Link to="/Cart">Cart(<CartCount numberOfCart={numberOfCart} />)</Link>
-                    </div>
-                </Navbar.Collapse>
+    <Navbar bg="light" expand="lg">
+      <Navbar.Toggle aria-controls="basic-navbar-nav" />
+      <Navbar.Collapse id="basic-navbar-nav" className={styles.navbarUuuu}>
+        <div className={styles.nav}>
+          <Nav className="1111">
```
Comment (justiceteam-it) [inline src/components/Navbar/Navbar.js:16]: what is it?

File: src/components/Navbar/Navbar.js [hunk 5]
```diff
@@ -1,29 +1,51 @@
...
+    //         </div>
+    //         <div className={styles.right}>
+    //           <Link to="/SignUp">SignUp</Link>
+    //           <Link to="/SignIn">SignIn</Link>
+    //           <Link to="/Cart">Cart</Link>
+    //         </div>
+    //       </Navbar.Collapse>
+    //     </div>
+    //   </Navbar>
+    // </div>
```
Comment (justiceteam-it) [inline src/components/Navbar/Navbar.js:47]: remove

File: src/layouts/Cart/styles.module.scss [hunk 6]
```diff
@@ -1,65 +1,6 @@
...
-  margin: 10px;
-}
-
-.info {
-  display: flex;
-  justify-content: flex-start;
   flex-direction: column;
-  border-radius: 5px;
-  border: 2px solid #e0e0e0;
+  margin: 50px!important;
```
Comment (justiceteam-it) [inline src/layouts/Cart/styles.module.scss:5]: don't use

File: src/layouts/SignIn/index.js [hunk 7]
```diff
@@ -33,7 +33,7 @@ const SignIn = () => {
                                             placeholder="Password"
                                         />
                                     </Form.Group>
-                                    <Button variant="primary" className="SignUpButton" type="submit">
+                                    <Button variant="primary" className={styles.SignUpButton} type="submit">
```
Comment (justiceteam-it) [inline src/layouts/SignIn/index.js:36]: lower case

File: src/layouts/SignUp/index.js [hunk 8]
```diff
@@ -51,7 +51,7 @@ const SignUp = props => {
                 <div className="SignUpForm">
                     <div className={styles.container}>
                         <Container>
-                            <h1 className="SignInHeading">Create an account</h1>
+                            <h1 className={styles.SignInHeading}>Create an account</h1>
```
Comment (justiceteam-it) [inline src/layouts/SignUp/index.js:54]: same

File: src/layouts/SignUp/index.js [hunk 9]
```diff
@@ -116,9 +116,11 @@ const SignUp = props => {
                                         {errors.confirmPassword}
                                     </Form.Control.Feedback>
                                     </Form.Group>
-                                    <Button variant="primary" className="SignUpButton" type="submit">
-                                        Sign Up
+                                  <div className={styles.button}>
+                                    <Button className={styles.SignUpButton} type="submit">
```
Comment (justiceteam-it) [inline src/layouts/SignUp/index.js:120]: same

File: src/styles.module.scss [hunk 10]
```diff
@@ -1,12 +1,13 @@
 .modal {
-  width: 1000px!important;
+  width: 1800px!important;
```
Comment (justiceteam-it) [inline src/styles.module.scss:2]: don't use

**Review by justiceteam-it — CHANGES_REQUESTED** [review]
please fix it. after merge

File: src/components/Navbar/Navbar.js (hunk 4 above)
Comment (danil-react) [reply src/components/Navbar/Navbar.js:16]: this is test class, i removed it

**Pull request merged**